crates/integration-tests/tests/gateway/extensions/field_resolver/url_subgraph_error.rs

```diff
@@ -0,0 +1,46 @@
+use integration_tests::{gateway::Gateway, runtime};
+
+use crate::gateway::extensions::field_resolver::StaticFieldResolverExt;
+
+#[test]
+fn resolver_extension_with_url_subgraph_returns_error() {
+    runtime().block_on(async move {
+        let result = Gateway::builder()
+            .with_toml_config(
+                r#"
+                [subgraphs.products]
+                url = "http://localhost:4001/graphql"
+                "#,
+            )
+            .with_subgraph_sdl(
+                "products",
+                r#"
+                extend schema
+                    @link(url: "static-1.0.0", import: ["@resolve"])
+
+                type Query {
+                    product(id: ID!): Product @resolve
+                }
+
+                type Product {
+                    id: ID!
+                    name: String!
+                    price: Float!
+                }
+                "#,
+            )
+            .with_extension(StaticFieldResolverExt::json(
+                r#"{"id": "1", "name": "Test Product", "price": 99.99}"#.into(),
+            ))
+            .try_build()
+            .await;
+
+        let error = result.unwrap_err();
+        let error_message = error.to_string();
+
+        assert!(
```

Comment:
you could have just used `insta::assert_snapshot(result.unwrap_err()` here, that's what I do at least

Reply:
fixed